_posts/2018-12-17-Devoxx-MA.md

```diff
@@ -0,0 +1,245 @@
+> the organisation behind Devoxx MA, decided to hold it in Marrakesh, in the lovely Palm Plaza Hotel on the 27th, 28th and 29th of November 2018.
+> Apart from the conference, xHub also offered a speakers trip to the speakers.
+> Four colleagues of Ordina JWorks: Yannick De Turck, Tim Ysewyn, Tom Van den Bulck and Maarten Casteels attended both the conference and the speakers trip.
+> Three out of them were also featured as speaker.
+> In this blog post we share our impressions and experiences.
+
+<img class="image fit" src="{{ '/img/devoxx-ma-2018/JWorks@DevoxxMA.jpg' | prepend: site.baseurl }}" alt="JWorks @ Devoxx MA" width="60%" />
+
+# Table of contents
+
+* [The Speakers Dinner](#the-speakers-dinner)
+* [VueJS Animation In Action by Charles-Philippe Bernard](#vuejs-animation-in-action-by-charles-philippe-bernard)
+* [MockK, The Idiomatic Mocking Framework For Kotlin by Yannick De Turck](#mockk-the-idiomatic-mocking-framework-for-kotlin-by-yannick-de-turck)
+* [Stream Processing Live Traffic Data with Kafka Streams by Tom Van den Bulck and Tim Ysewyn](#stream-processing-live-traffic-data-with-kafka-streams-by-tom-van-den-bulck-and-tim-ysewyn)
+* [Applying (D)DDD and CQ(R)S to Cloud Architectures with Spring Boot and Docker by Benjamin Nothdurft](#applying-dddd-and-cqrs-to-cloud-architectures-with-spring-boot-and-docker-by-benjamin-nothdurft)
+* [Resiliency for 140 PB Cluster by Meriam Lachkar](#resiliency-for-140-pb-cluster-by-meriam-lachkar)
+* [La Keynote De La Nouvelle Generation by Saskia and Lois](#la-keynote-de-la-nouvelle-generation-by-saskia-and-lois)
+* [The Speakers Trip in Essaouira](#the-speakers-trip-in-essaouira)
+
+## The Speakers dinner
+The evening before the conference started, we had the speakers dinner sponsored by [Lightbend](https://www.lightbend.com){:target="_blank" rel="noopener noreferrer"}.
+The dinner was at restaurant [Kasar El Hamra](http://www.restaurant-ksarelhamra.net){:target="_blank" rel="noopener noreferrer"} in the center of Marrakesh.
+On our way to the center we got to experience the Moroccan traffic which seemed to be pretty chaotic with all the cars and motorcycles zipping around, evading each other on the streets.
+Once at the restaurant we got treated with one Moroccan dish after the other.
+Each of them truly delicious. 
+
+<img class="image fit" src="{{ '/img/devoxx-ma-2018/speakers-dinner.jpg' | prepend: site.baseurl }}" alt="Speakers Dinner" width="60%" />
+
+## VueJS animation in action by [Charles-Philippe Bernard](https://twitter.com/charlyouki){:target="_blank" rel="noopener noreferrer"}
+
+<span class="image left"><img class="p-image" alt="Charles-Philippe Bernard" src="/img/devoxx-ma-2018/speaker-charles-philippe.png"></span>
+
+Charles-Philippe explained why he loves [VueJS](https://vuejs.org){:target="_blank" rel="noopener noreferrer"} so much.
+He criticised Angular and React because of the companies behind the frameworks as well as the multiple variations of utilities and plugins. 
```

Comment:
```suggestion
He criticised Angular and React because of the companies behind the frameworks, as well as the multiple variations of utilities and plugins. 
```